pennylane/tape/tapes/tape.py

```diff
@@ -224,15 +220,34 @@ class QuantumTape(AnnotatedQueue):
     [0.56]
     >>> tape.get_parameters(trainable_only=False)
     [0.56, 0.543, 0.133]
+
+
+    When using a tape with ``embed=False``, that tape will not be embedded in a parent tape context.
+
+    .. code-block:: python
+
+    with qml.tape.QuantumTape() as tape1:
+        with qml.tape.QuantumTape(embed=False) as tape2:
+            qml.RX(0.123, wires=0)
+
+    Here, tape2 recods the RX gate, but tape1 doesn't record tape2.
+
+    >>> tape1.operations
+    []
+    >>> tape2.operations
+    [RX(0.123, wires=[0])]
```

Comment:
Ideal here would be to have:
```
>>> tape1.operations
[qml.tape.QuantumTape object at 0xDEADBEEF]
>>> tape2.operations
[RX(0.123, wires=[0])]
```

Reply:
(this has nothing to do with adjoint, just a comment on how I feel nesting should look)

Reply:
> Ideal here would be to have:
> ...
> Seems weird that tape1.operations is "empty" when it contains tape2

@co9olguy this is currently how tape nesting works:

```pycon
>>> with qml.tape.QuantumTape() as tape1:
...     with qml.tape.QuantumTape() as tape2:
...         qml.RX(0.123, wires=0)
>>> tape1.operations
[<QuantumTape: wires=[0], params=1>]
>>> tape2.operations
[RX(0.123, wires=[0])]
```

The `embed=True` changes this behaviour; `tape2` is *not queued*, and `tape1` remains empty. Actually, now that I think about it, you can consider `embed` in this context the same as `do_queue`.

@Thenerdstation any reason for choosing a different name here?